tools/apiview/parsers/swagger-api-parser/reviewGists/SwaggerReviewNonJsonView.md

```diff
@@ -0,0 +1,212 @@
+###  `/deviceupdate/{instanceId}/updates`
+
+<details>
+<summary>POST</summary>
+
+``` yaml
+  DeviceUpdate_ImportUpdate
+  Import new update version.
+  x-ms-long-running-operation: true
+  
+  Parameters:
+    PathParameters:
+      instanceId: string
+    QueryParameters:
+      action: string
+      api-verison: string
+    BodyParameters:
+      updateToImport:
+        ImportUpdateInput
+          array<object>
+  Responses:
+    202:
+      description: Accepted update import request; backgroud operation location to track status is specified in Operation-Location response headers
+      Schema:
+        Update
+          updateId:
+            UpdateId
+              provider: string
+              name: string
+              version: string
+           description: string
+           frendlyName: string
+           isDeployable: boolean
+           updateType: string
+           installedCriteria: string
+           compatibility: array<object>
```

Comment:
it might be a good idea to include a reference to an `object` and ask the architects if they want us to be explicit here about the content of the object, or to make the `object` clickable and jump to a definition section or something